packages/jest/test/react-enzyme.test.js

```diff
@@ -191,6 +204,25 @@ describe('enzyme', () => {
     cases
   )
 
+  test('child containing css array property', () => {
```

Comment:
is this the correct test title? no css prop here holds an array

Reply:
updated, as noted the titles got switched